updateVersions.sc

```diff
@@ -123,7 +123,7 @@ def main(updatedImage: String, updatedImageReleaseNote: String): Unit = {
                 val firstSplit = s.split(":")
                 val splited = firstSplit(1).split("\\.")
                 s"${firstSplit(0)}:${splited(0)}.${splited(1)}.${splited(2).toInt + 1}\n"
-              } else if(s.contains(" AS ")) {
+              } else if(s.contains(" AS ") && imagesToUpdate.contains("terra-jupyter-python")) {
```

Comment:
I think this won't be valid if the image getting updated is `terra-jupyter-base`

Reply:
hmm i have a separate branch where i made a change to `terra-jupyter-base` so I can test there

Reply:
it defnitely won't work because when you change terra-jupyter-base, both terra-jupyer-python and terra-jupyter-r will change...so the original logic is fine in that case. you need to somehow make this conditional depends on whether the "base" image is `terra-jupyter-r`